committee-steering/sig-governance.md

```diff
@@ -0,0 +1,629 @@
+they own within the SIG.  Members are expected to help other
+members in the SIG (e.g. transfer knowledge, help debug, assist with
+operational tasks, answer questions, etc).
+
+Membership is provisional until the member has shown sustained
+contributions for 3 months.  Provisional members may provide
+feedback in decision making, but are not deciders in the decision
+itself.
+
+## Processes and tools
+
+The following section outlines various processes used by the SIG.
+
+### Decision making within the SIG
+
+It is critical to a SIG that the decision making processes is healthy.
+A good decision making process has the following properties:
+
+- Decision making process is unambiguously understood by everyone
+  involved
+  - Clear definition of the set of individuals responsible for making
+    decisions
+  - Clear definition of the decision making process used by those
+    individuals
+- Decisions are resolved instead of languishing and defaulting to no
+  decision
+- Decisions are a commitment and won't waffle after having been made
+
+#### Who, What, Where, When, How
+
+**Note:** Proposals should be sent before or early in the release
+cycle they are targeted to.  The SIG may decide to defer discussing
+or accepting proposals during stabilization periods when the SIG
+is focused on ensuring the existing functionality is stable and
+ready to be released.
+
+In cases where a proposal author is seeking acceptance from the
+community it is important to have a clearly defined process for
+communication and resolution of the proposal.
+
+While it is not necessary for all decisions to go through the lazy
+consensus through the full SIG, it is important that such a process
+exists for decisions that may affect many members of the SIG.
+
+When using [lazy consensus][lazy-consensus], all proposals must be
+published to a visible location.  For acceptance to be given be
+implicitly (as it is in lazy consensus models), all involved parties
+must be made aware of the proposal.  While it is the responsibility
+of involved parties to watch for new proposals and read them in a
+timely manner, it is the responsibility of the SIG governance to
+ensure that where and how are well defined.
+
+Authors of proposals should generally allow at least 5 business days
+before considering a proposal accepted.  Large, complex, or impactful
+proposals should allow more time than normal and should ensure that
+a quorum of area OWNERs are present at the time the proposal is
+made (e.g. not on vacation).  Area owners may defer discussion of
+a proposal if for some reason it is not a good time.
+
+In order to avoid [Warnock's Dilemma][warnocks-dilemma], it is
+recommended at least one active member respond to a proposal before
+it is accepted.
+
+> **Note:** To those forking this template - consider changing the
+  following text to only include the tools actively used by your SIG.
+
+In SIG ~~Foo~~, proposals are made using a commonly used tool
```

Comment:
I would suggest the KEP process be followed, and that we change/improve that to work the way we want, as necessary.

General decision-making procedure recommendations maybe should be split out to a separate document. The general "lazy consensus" definition isn't sufficient, in my experience.